rustworkx-core/src/generators/lollipop_graph.rs

```diff
@@ -15,19 +15,18 @@ use std::hash::Hash;
 use petgraph::data::{Build, Create};
 use petgraph::visit::{Data, GraphProp, NodeIndexable};
 
-use super::utils::pairwise;
+use super::utils::{get_num_nodes, pairwise};
 use super::InvalidInputError;
 
-/// Generate a lollipop graph
+/// Generate an undirected lollipop graph where a complete graph is connected to a
+/// path.
 ///
-/// Arguments:
+/// .. note::
```

Comment:
I don't think this syntax will work in rustdoc since it's markdown based. I'm not sure what the correct syntax is off the top of my head, we'll have to check the docs.

Reply:
This passes the tests. Is there somewhere this would fail and not be tested?

Reply:
Thanks for that info. I went back through all the docs in core and I think everything should be clean now. I removed all the markdown and made things prettier. Turns out the trick for displaying Ascii is to use `text` instead of `rust` for code-like sections.
Should be done in https://github.com/Qiskit/rustworkx/pull/784/commits/63d2307d1fd505b22ef802a407d05b26b27e08a9 and https://github.com/Qiskit/rustworkx/pull/784/commits/b0e4dfc044c1db120755be7d4a4c82b6c574b412.